exampleCourse/serverFilesCourse/templates/collapsible_section.mustache

```diff
@@ -4,8 +4,8 @@
   <div class="card-header" style="cursor: pointer">
     <div
       class="card-title d-flex justify-content-between"
-      data-toggle="collapse"
-      data-target="#collapse-{{uuid}}"
+      data-bs-toggle="collapse"
+      data-bs-target="#collapse-{{uuid}}"
```

Comment:
@eliotwrobson this example isn't accessible, as it can't be interacted with via a keyboard and lacks necessary `aria-*` attributes.

Reply:
This is now tracked in https://github.com/PrairieLearn/PrairieLearn/issues/11444.